sbin/load-celeborn-env.sh

```diff
@@ -69,3 +69,24 @@ if [ "$CELEBORN_PID_DIR" = "" ]; then
   export CELEBORN_PID_DIR="${CELEBORN_HOME}/pids"
 fi
 
+# jemalloc memory allocator is enabled by default, you may set DISABLE_JEMALLOC to true in celeborn-env.sh if it's not as you wish.
```

Comment:
Not every OS has jemalloc installed, and now the start and stop commands will have multiple WARNING.

We can disable it by default, then enable it in `celeborn-env.sh`, and add instructions to the website or README.

```bash
./sbin/start-master.sh
```

```
WARNING: attempted to load jemalloc from /usr/lib/arm64-linux-gnu/libjemalloc.so and /usr/lib/x86_64-linux-gnu/libjemalloc.so but the library couldn't be found. glibc will be used instead.
WARNING: attempted to load jemalloc from /usr/lib/arm64-linux-gnu/libjemalloc.so and /usr/lib/x86_64-linux-gnu/libjemalloc.so but the library couldn't be found. glibc will be used instead.
starting org.apache.celeborn.service.deploy.master.Master
```

Reply:
sound reasonable. how about 

1) make `PREFER_JEMALLOC` and `JEMALLOC_PATH` configurable in `celeborn-env.sh`
2) disable `PREFER_JEMALLOC` by default, at least for `branch-0.3`, we may enable it by default in the future
3) consider the adoption rate of Linux distribution, when `PREFER_JEMALLOC=true` and `JEMALLOC_PATH` is absent, search the default installed path of jemalloc libraries on Debian and CentOS 7